src/Razor/Microsoft.AspNetCore.Razor.Tools/src/RewriteCssCommand.cs

```diff
@@ -257,6 +281,35 @@ protected override void VisitAtDirective(AtDirective item)
             }
         }
 
+        private class EnsureNoImports : Visitor
+        {
+            private readonly string _filePath;
+            private readonly string _sourceText;
+            private readonly List<RazorDiagnostic> _diagnostics;
+
+            public EnsureNoImports(string filePath, string sourceText, ComplexItem root, List<RazorDiagnostic> diagnostics) : base(root)
+            {
+                _filePath = filePath;
+                _sourceText = sourceText;
+                _diagnostics = diagnostics;
+            }
+
+            protected override void VisitAtDirective(AtDirective item)
+            {
+                if (item.Children.Count >= 2
+                    && item.Children[0] is TokenItem firstChild
+                    && item.Children[1] is TokenItem secondChild
+                    && firstChild.TokenType == CssTokenType.At
+                    && string.Equals(secondChild.Text, "import", StringComparison.OrdinalIgnoreCase))
```

Comment:
Just had to look up case sensitiveness of CSS, didn't realize it was case insensitive interesting...